android/modules/network/src/java/ti/modules/titanium/network/TiHTTPClient.java

```diff
@@ -245,7 +246,7 @@ private void handleResponse(HttpURLConnection connection) throws IOException
 				Log.d(TAG, "Content length: " + contentLength, Log.DEBUG_MODE);
 				int count = 0;
 				long totalSize = 0;
-				byte[] buf = new byte[4096];
+				byte[] buf = new byte[8192];
```

Comment:
Extract the magic constant as a final static at top so it's easier to tweak the buffer size?

(in fact looking at this file it looks like we have a number of magic constants around buffers/sizes: 512 bytes for progress notifications/entity data buffer; 8MB for blob/file I/O buffer; 512Kb for dumping response to file first rather than keep in-memory)

Reply:
It's not worth changing to a constant if it's only used in 1 place in the code. It's better to keep it simple.

Yes, there are other place in this class where there is duplication, but that's a refactoring for another day.